wolfcrypt/src/ed25519.c

```diff
@@ -406,63 +463,34 @@ int wc_ed25519_verify_msg_ex(const byte* sig, word32 sigLen, const byte* msg,
     ge_p2  R;
 #endif
     int    ret;
-    int    devId = INVALID_DEVID;
-    wc_Sha512 sha;
 
     /* sanity check on arguments */
-    if (sig == NULL || msg == NULL || res == NULL || key == NULL ||
-                                         (context == NULL && contextLen != 0)) {
+    if (sig == NULL || res == NULL || key == NULL) {
         return BAD_FUNC_ARG;
     }
 
     /* set verification failed by default */
     *res = 0;
 
     /* check on basics needed to verify signature */
-    if (sigLen != ED25519_SIG_SIZE || (sig[ED25519_SIG_SIZE-1] & 224))
-        return BAD_FUNC_ARG;
-
-#ifdef WOLF_CRYPTO_CB
```

Comment:
Looks like the recent ED25519 crypto callback support was removed?

Reply:
thank you for catching this!  completely inadvertent, and now restored with proper error checking.